php/class-wp-cli.php

```diff
@@ -982,6 +982,10 @@ public static function print_value( $value, $assoc_args = array() ) {
 	 * @return string
 	 */
 	public static function error_to_string( $errors ) {
+		if ( ! is_string( $errors ) && ! $errors instanceof \WP_Error ) {
+			throw new InvalidArgumentException( sprintf( 'Unsupported argument type passed to WP_CLI::error_to_string(): %s', gettype( $errors ) ) );
+		}
```

Comment:
This now breaks the code at the bottom that is meant to allow the method to handle exceptions as well.

Instead of having a check here, it should just have the `throw` statement at the very end of the method as a sort of catch-all.

Reply:
Ahhh!! true.